augur/export_v2.py

```diff
@@ -877,6 +887,9 @@ def register_parser(parent_subparsers):
                                  help="delimiters to accept when reading a metadata file. Only one delimiter will be inferred.")
     optional_inputs.add_argument('--metadata-id-columns', default=DEFAULT_ID_COLUMNS, nargs="+",
                                  help="names of possible metadata columns containing identifier information, ordered by priority. Only one ID column will be inferred.")
+    optional_inputs.add_argument('--metadata-columns', nargs="+",
```

Comment:
Any functionality we support via command line arguments should be available via the auspice-config JSON as well, as the former is a subset of the latter's capabilities.

Reply:
Hmm, this flag relates to the metadata input/export but not really Auspice control settings. Not sure this fits in the auspice-config JSON?

Reply:
I see it as controlling what data is exported, similar to filters & colours. The (original) intention when we started adding command line arguments for options in the config JSON was for auspice-config to be all you need to control what data the JSON contains.

Reply:
Okay, that makes sense. I'll move up the flag to be part of the `config` args group too then.

Reply:
Auspice config schema update and changes to support the new property added in bdb2b2c66d2322a49d0484a09006279500819b3b